pontoon/insights/static/js/insights_charts.js

```diff
@@ -40,6 +40,60 @@ var Pontoon = (function (my) {
         Chart.defaults.global.datasets.bar.barPercentage = 0.7;
         Chart.defaults.global.datasets.bar.categoryPercentage = 0.7;
       },
+      // Legend configuration doesn't allow for enough flexibility,
+      // so we build our own legend
+      // eslint-disable-next-line no-unused-vars
+      customLegend: function (chart) {
+        return function (chart) {
+          function renderLabels(chart) {
+            return chart.data.datasets
+              .map(function (dataset) {
+                var disabled = dataset.hidden ? 'disabled' : '';
+                var color = dataset.borderColor || dataset.backgroundColor;
+
+                return (
+                  '<li class="' +
+                  disabled +
+                  '">' +
+                  '<i class="icon" style="background-color:' +
+                  color +
+                  '"></i>' +
+                  '<span class="label">' +
+                  dataset.label +
+                  '</span>' +
+                  '</li>'
+                );
```

Comment:
This is why we have template literals. Same comment applies to other string concatenations here. If the lines get too long, you can use `\` to escape newlines that are then ignored.
```suggestion
                return (
                  `<li class="${disabled}"><i class="icon" style="background-color:${color}"></i><span class="label">${dataset.label}</span></li>`
                );
```